.prettierignore

```diff
@@ -0,0 +1 @@
+src/tests/ReactSixteenAdapter.js
```

Comment:
Isn't this now `tests/ReactSixteenAdapter.js`?

Reply:
True! Will fix